controller/search_blackbox_test.go

```diff
@@ -411,6 +413,10 @@ func (s *searchControllerTestSuite) TestSearchByWorkItemTypeGroup() {
 	s.T().Run(http.StatusText(http.StatusOK), func(t *testing.T) {
 		// given
 		fxt := tf.NewTestFixture(t, s.DB,
+			tf.Spaces(1, func(fxt *tf.TestFixture, idx int) error {
+				fxt.Spaces[idx].SpaceTemplateID = spacetemplate.SystemLegacyTemplateID
```

Comment:
@jarifibrahim why does the space have to use a legacy template?

Reply:
Because all the test workitems created here https://github.com/fabric8-services/fabric8-wit/pull/2115/files/43ac08e2c1416d7bfbe572bf3e921d9e5946c320#diff-a2bd8cd90ce50fc5463d8b3539620c92R464 belong to legacy template. If the space doesn't belong to `legacy template` then the test would fail.

Reply:
@jarifibrahim Hm, so before it worked and now it doesn't since you disallowed creation of `system.bug` in a random space that isn't based on the template that has these WITs, correct?

Reply:
That's correct